spring-session-data-mongodb/src/main/java/org/springframework/session/data/mongo/MongoSession.java

```diff
@@ -66,7 +67,7 @@ public class MongoSession implements Session {
 	private Map<String, Object> attrs = new HashMap<>();
 
 	public MongoSession() {
-		this(MongoIndexedSessionRepository.DEFAULT_INACTIVE_INTERVAL);
+		this(MapSession.DEFAULT_MAX_INACTIVE_INTERVAL_SECONDS);
 	}
 
 	public MongoSession(long maxInactiveIntervalInSeconds) {
```

Comment:
Should this use java.time as well? It seems like there are other locations where int or long are still being used as well. I know that the annotations need to use primitives, but I think the setters on the configuration should still be java.time and we should convert it.